Sources/MapboxCoreNavigation/NavigationEventsManager.swift

```diff
@@ -56,6 +56,12 @@ open class NavigationEventsManager {
         }
     }
     
+    /**
+     Optional application metadata that that can help Mapbox more reliably diagnose problems that occur in the SDK.
+     For example, you can provide your application’s name and version, a unique identifier for the end user, and a session identifier.
```

Comment:
Should we provide the constants for the keys mentioned in this docs note?

Reply:
As in the naming? Hm...I don't think it's necessary. We don't want to include this value in the event if it isn't set. Do you think it could cause an issue with ingestion?

Reply:
I'm talking about `you can provide your application’s name and version, a unique identifier for the end user, and a session identifier`. This line says "I can" but I don't know what key to use. If we want this to be internal, maybe it's better not to document this property? Otherwise, I would expect at least to mention dictionary keys available to developers.